### PR TITLE
[cleanup] [trivial] Return bool from Expression::isLvalue

### PR DESCRIPTION
**Pull request by yebblies**

### CODE REVIEW

File: src/expression.c [hunk 1]
```diff
@@ -6762,9 +6762,9 @@ Expression *BinAssignExp::semantic(Scope *sc)
     return ((BinExp *)e)->reorderSettingAAElem(sc);
 }
 
-int BinAssignExp::isLvalue()
+bool BinAssignExp::isLvalue()
 {
-    return 1;
+    return tru;
```
Comment (AndrejMitrovic) [inline src/expression.c]: * `true`

Comment (AndrejMitrovic) [inline src/expression.c]: Sadly D doesn't have -gangsta mode yet.

Comment (yebblies) [inline src/expression.c]: Soon.

**Comment by yebblies** [comment]
Updated.

**Comment by AndrejMitrovic** [comment]
Auto-merge toggled on

**Pull request merged**